Containerfile

```
@@ -37,3 +37,10 @@ RUN --mount=type=bind,target=/run/src --mount=type=secret,id=yumrepos,target=/et
   /run/src/scripts/apply-manifest /run/src/packages-openshift.yaml && \
   find /usr -name '*.pyc.bak' -exec sh -c 'mv $1 ${1%.bak}' _ {} \; && \
   ostree container commit
+
+FROM build as metadata
+RUN --mount=type=bind,target=/run/src /run/src/scripts/generate-metadata
+
+FROM build
+COPY --from=metadata /usr/share/openshift /usr/share/openshift
+LABEL io.openshift.metalayer=true
```

I'm slightly confused.. I thought each operation in a Containerfile made a new layer anyway.. so how is what you have here different than not doing a multi-stage build and just doing:

```suggestion
RUN --mount=type=bind,target=/run/src /run/src/scripts/generate-metadata
LABEL io.openshift.metalayer=true
```

---

Running the script itself adds more than just the metadata. It regenerates Python cache, it touches the rpmdb (just querying will change the timestamp), etc... Doing it this way ensures there's only _just_ the metadata in there.